SpinCore_pp/ppg/generic.py

```diff
@@ -25,7 +25,7 @@ def generic(
     adcOffset,
     carrierFreq_MHz,
     nPoints,
-    acq_time_ms,
+    time_per_segment_ms,
```

Comment:
you're going to have to change the name of this guy in all the pulse sequences that use ppg/generic